eng/common/scripts/get-diff-changes-from-PR.ps1

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Lets put this script under eng/common/scripts/helpers/ and name it "git-helpers.ps1.

Reply:
I am thinking people who use 'common.ps1' will import the function directly. The script is needed, and I agree is kind of helper file we can put under `helper`